test/test_pebble.py

```diff
@@ -3185,6 +3185,73 @@ def test_get_notices_filters(self, client: MockClient):
             ('GET', '/v1/notices', query, None),
         ]
 
+    def test_get_identities(self, client: MockClient):
+        client.responses.append({
+            'result': {
+                'alice': {'access': 'admin', 'local': {'user-id': 42}},
+                'web': {'access': 'metrics', 'basic': {'password': 'hashed password'}},
+            },
+            'status': 'OK',
+            'status-code': 200,
+            'type': 'sync',
+        })
+
+        identities = client.get_identities()
+        assert len(identities) == 2
+        assert identities['alice'] == pebble.Identity.from_dict({
+            'access': 'admin',
+            'local': {'user-id': 42},
+        })
+        assert identities['web'] == pebble.Identity.from_dict({
+            'access': 'metrics',
+            'basic': {'password': 'hashed password'},
+        })
+
+        assert client.requests == [
+            ('GET', '/v1/identities', None, None),
+        ]
+
+    def test_replace_identities(self, client: MockClient):
+        client.responses.append({
+            'status': 'OK',
+            'status-code': 200,
+            'type': 'sync',
+        })
+        identities: typing.Dict[str, typing.Union[pebble.IdentityDict, pebble.Identity, None]] = {
+            'web': {'access': 'metrics', 'basic': {'password': 'hashed password'}},
```

Comment:
Would be good to have one entry that's an `Identity`, one that's an `IdentityDict`, and one that's `None`, to test each of those cases.